Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -557,30 +552,78 @@ public bool RegisterService(Type type, IMixedRealityService service)
                 return false;
             }
 
-            if (IsCoreSystem(type))
+            return RegisterServiceInternal(type, service);
+        }
+
+        /// <summary>
+        /// Add a new service to the Mixed Reality Toolkit active service registry.
+        /// </summary>
+        /// <typeparam name="T">The interface type for the system to be registered.</typeparam>
+        /// <param name="concreteType">The concrete type to instantiate.</param>
+        /// <param name="args">Optional arguments used when instantiating the concrete type.</param>
+        /// <returns></returns>
+        public bool RegisterService<T>(Type concreteType, params object[] args)
+        {
+            if (ActiveProfile == null) { return false; }
+
+            if (concreteType == null)
+            {
+                Debug.LogError("Unable to register a service with a null concrete type.");
+                return false;
+            }
+
+            if (!typeof(IMixedRealityService).IsAssignableFrom(concreteType))
+            {
+                Debug.LogError($"Unable to register the {concreteType.GetType()} service. It does not implement the IMixedRealityToolkitService interface.");
```

Comment:
The interface checked above is `IMixedRealityService`, but this comment references `IMixedRealityToolkitService`. I think the comment needs to be updated.